### PR TITLE
#68 Implemented homepage design based on mockup

### LINKED ISSUE #68: #12.1 Homepage
As a user, I want to be able to access the home page, so that I can encounter and view the first page of the website.

Story Points: 2
MoSCoW: Must
Business Value: L
Risk: Low

Acceptance Criteria

- Upon landing on the website, the user is looking at the home page

Child of #67

### PR DESCRIPTION
**Pull request by janong24**
Homepage design complete.
Graphical assets uploaded.

Closes #68

### CODE REVIEW

**Comment by kennyphan100** [comment]
Could you possibly take a screenshot of the homepage and post it in the description of the PR, just so that we can see what we are working with. If you are busy, then don't worry about it.

**Comment by janong24** [comment]
@kennyphan100 Yeah, for sure. It's an exact replica of the Figma.

https://drive.google.com/file/d/1jvQ-LMVo0lPDRqUbh-pMTJZWlYTq8vFv/view?usp=share_link

**Comment by kennyphan100** [comment]
> @kennyphan100 Yeah, for sure. It's an exact replica of the Figma.
> 
> https://drive.google.com/file/d/1jvQ-LMVo0lPDRqUbh-pMTJZWlYTq8vFv/view?usp=share_link

It looks amazing, well done!

**Review by CHADBUCK13 — CHANGES_REQUESTED** [review]
Looks good! I have a feeling you may not merged the main branch into your branch before comitting though, please see the comments.

File: synapsis/src/main/resources/templates/fragments/navbar.html [hunk 1]
```diff
@@ -32,53 +31,4 @@
             </div>
         </div>
     </nav>
```
[the remark this answers is not in the document]
Comment (janong24) [reply synapsis/src/main/resources/templates/fragments/navbar.html:33]: Thanks for pointing these out! I think these changes came after my PR. I'll resolve these and resend a request.

[the remark this answers is not in the document]
Comment (janong24) [reply synapsis/src/main/resources/templates/fragments/navbar.html]: I've made the necessary changes. Please take a look when you have the time @CHADBUCK13. Thanks!

**Review by Chr728 — APPROVED** [review]

**Review by CHADBUCK13 — APPROVED** [review]

**Pull request merged**